parquet-hadoop/src/main/java/parquet/filter2/statisticslevel/StatisticsFilter.java

```diff
@@ -118,7 +130,12 @@ private boolean hasNulls(ColumnChunkMetaData column) {
       return false;
     }
 
-    Statistics<T> stats = columnChunk.getStatistics();
+    if (isAllNulls(columnChunk)) {
+      // there is no min max, there is nothing
+      // else we can say about this chunk, we
+      // cannot drop it.
+      return false;
+    }
```

Comment:
this filter is notEq filter , where we are looking for columnValue != (some non null value), so  any row with null column should be returned and cannot be dropped.

Also I think the above check is redundant anyways, since we have a hasNulls() check above this one, so if there are any nulls, we will be returning 'false' and this check would never be reached.